recipe/build.sh

```diff
@@ -27,7 +27,11 @@ rm -f build/pkgs/boost/spkg-configure.m4
 
 rm $PREFIX/bin/$HOST-pkg-config
 make configure
-./configure --prefix="$PREFIX" --with-python="$PYTHON"
+./configure \
+  --prefix="$PREFIX" \
+  --with-python="$PYTHON" \
+  --with-system-pkgconf=no \
```

Comment:
Please remove this

Reply:
Done. There are two type of errors, pkgconfig module not found and in some architectures, sirocco not found

Reply:
Do you have any ideas how to solve it? I wonder if something must be changed in sirocco to work with other architectures. 

I have no clue how to deal with the problem of the pkgconfig module in the test.